librz/analysis/fcn.c

```diff
@@ -1373,8 +1402,44 @@ static int fcn_recurse(RzAnalysis *analysis, RzAnalysisFunction *fcn, ut64 addr,
 	return ret;
 }
 
-RZ_API int rz_analysis_fcn_bb(RzAnalysis *analysis, RzAnalysisFunction *fcn, ut64 addr, int depth) {
-	return fcn_recurse(analysis, fcn, addr, analysis->opt.bb_max_size, depth - 1);
+RZ_API bool rz_analysis_task_item_new(RzAnalysis *analysis, RzVector *tasks, RzAnalysisFunction *fcn, RzAnalysisBlock *block, ut64 address) {
```

Comment:
For all new API functions please add a Doxygen description: https://github.com/rizinorg/rizin/blob/dev/DEVELOPERS.md#documentation